CONTRIBUTING.md

```diff
@@ -1,193 +1,169 @@
+5. **Commit your changes** by clicking the _Propose changes_ button.
+6. **Create a Pull Request (PR)** by clicking _Compare & pull request_.
+7. **Review process**: Someone from the core team will review your changes. This may take a few hours to a few days.
 
-In case you want to make some bigger changes, like adding a new article, or edit multiple files, you should fork the project in the traditional way, create a branch, modify the files in the Github UI or locally on your computer, and create a pull-request.
-If you are unfamiliar with the workflow, read [Step-by-step guide to contributing on GitHub](https://www.dataschool.io/how-to-contribute-on-github/).
+### Making Larger Changes
 
-If you're making a new article or moving existing one to a different place, please make sure that your changes are reflected in
+If you’re planning to make more significant changes, such as adding new articles or modifying multiple files:
 
-- The list of all articles in [navigation.md](https://github.com/cp-algorithms/cp-algorithms/blob/main/src/navigation.md);
-- The list of new articles in [README.md](https://github.com/cp-algorithms/cp-algorithms/blob/main/README.md) (if it is a new article).
+- **Fork the project** using the traditional Git workflow (create a branch for your changes).
+- **Edit files locally or in the GitHub UI**.
+- **Submit a pull request** with your updates.
 
-## Syntax
+For help with this workflow, check out this helpful guide: [Step-by-step guide to contributing on GitHub](https://www.dataschool.io/how-to-contribute-on-github/).
 
-We use [Markdown](https://daringfireball.net/projects/markdown) for the articles, and use the [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) to render the Markdown articles into HTML.
+### Updating Indexes
 
-For advanced Markdown features of Material for MkDocs see their [reference pages](https://squidfunk.github.io/mkdocs-material/reference/formatting), like:
+When you add new articles or reorganize existing ones, be sure to update the following files:
 
-- [Math formulas with MathJax](https://squidfunk.github.io/mkdocs-material/reference/mathjax/#usage)
-  Notice that you need to have an empty line before and after a `$$` math block.
-- [Code blocks](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#usage) for code snippets.
-- [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage) (e.g. to decor theorems, proofs, problem examples).
-- [Content tabs](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#usage) (e.g. for code examples in several languages).
-- [Data tables](https://squidfunk.github.io/mkdocs-material/reference/data-tables/#usage).
+- **[navigation.md](https://github.com/cp-algorithms/cp-algorithms/blob/main/src/navigation.md)**: Update the list of all articles.
+- **[README.md](https://github.com/cp-algorithms/cp-algorithms/blob/main/README.md)**: Add new articles to the main project overview.
```

Comment:
The second line is better because it clearly instructs contributors to update the existing list of new articles, ensuring clarity in the required action.